addons/osio-addon/pom.xml

```diff
@@ -32,6 +32,21 @@
       <artifactId>launcher-service-openshift-impl</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-validator</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.el</groupId>
+      <artifactId>javax.el-api</artifactId>
```

Comment:
Isn't this dependency transitively resolved from org.glassfish.web:javax.el?

Reply:
yeah fixed